doc/examples/parallax/lib/main_game.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flame/components.dart';
+import 'package:flame/flame.dart';
+import 'package:flame/game.dart';
+import 'package:flame/parallax.dart';
+import 'package:flame/extensions.dart';
+import 'package:flutter/material.dart';
+
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  await Flame.device.fullScreen();
+  runApp(
+    GameWidget(
+      game: MyGame(),
+    ),
+  );
+}
+
+class MyGame extends Game {
```

Comment:
Maybe we should add a comment saying that this one is using `Game` and that the easier way is using `BaseGame`? Seems to be a lot of people lately that accidentally extends `Game` instead of `BaseGame`, and that could probably originate from them copying things from the examples.
We could also just not have an example with `Game`?

Reply:
I kind of did this example to test my changes and make sure that the "plain" feature was working, I don't think we should have a strong opinion, like saying that `BaseGame` is easier than `Game`, but I will fashion a comment and you can help me improve that comment

Reply:
But most of our features are building on `BaseGame`?
I think we should have an opinion towards that, and then offer `Game` as an alternative if the implementer wants to do something from the bottom up.

Reply:
That was in a more general sense, I think a comment on here will work fine!

Reply:
I have added a simple comment, let me know if that is enough, or if we should be more "harsh"

Reply:
I think new users coming here doesn't know what FCS is, maybe add something about it not using `BaseGame`?
LIke:
/// This examples serves to test the Parallax feature outside *of the*/// This examples serves to test the Parallax feature outside
/// Flame Component System (FCS), use the other files in this folder
/// for examples on how to use parallax with FCS
/// Flame Component System (FCS), use the other files in this folder
/// for examples on how to use parallax with FCS.
/// FCS is only used when you extend BaseGame, not Game,
/// like we do in this example.

This should of course be clear from our docs too, but I think a lot of people just dive directly into the examples without looking at the docs. What do you think?

Reply:
Sorry, your comment example has some duplicated lines, I am a little confused.

About adding more docs about the FCS on BaseGame, I sure think we should do that, but I think this should be done on another PR, what do you think?

Reply:
Oops, must have pasted it twice somehow, edited it now.
Absolutely, didn't mean that the docs for FCS should be added in this PR, just thinking out loud. :)

Reply:
Updated the comment! :D

Reply:
Approved!